tests/rest/client/test_directory.py

```diff
@@ -32,15 +36,19 @@ class DirectoryTestCase(unittest.HomeserverTestCase):
         room.register_servlets,
     ]
 
-    def make_homeserver(self, reactor, clock):
+    def make_homeserver(self, reactor: MemoryReactor, clock: Clock) -> HomeServer:
         config = self.default_config()
         config["require_membership_for_aliases"] = True
 
         self.hs = self.setup_test_homeserver(config=config)
 
         return self.hs
 
-    def prepare(self, reactor, clock, homeserver):
```

Comment:
FWIW It might be easy to do a PR that modifies this (and `make_homeserver`) with `sed` + `black`. Not sure if it would make things pass or not though.